pom.xml

```diff
@@ -377,6 +378,41 @@
         </executions>
       </plugin>
 
+      <!-- Flatten pom plugin -->
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>flatten-maven-plugin</artifactId>
+        <version>${plugin.version.flatten}</version>
+        <configuration combine.children="append">
```

Comment:
🔧 The `combine.children` is not necessary here, because this configuration is not overwriting or adding to an inherited configuration for the flatten plugin. It's only defined here. The attribute can be removed.